src/reports/TripReportPage.jsx

```diff
@@ -189,16 +200,21 @@ const TripReportPage = () => {
             <TableBody>
               {!loading ? items.map((item) => (
                 <TableRow key={item.startPositionId}>
-                  <TableCell className={classes.columnAction} padding="none">
-                    {selectedItem === item ? (
-                      <IconButton size="small" onClick={() => setSelectedItem(null)}>
-                        <GpsFixedIcon fontSize="small" />
-                      </IconButton>
-                    ) : (
-                      <IconButton size="small" onClick={() => setSelectedItem(item)}>
-                        <LocationSearchingIcon fontSize="small" />
+                  <TableCell className={classes.columnAction}>
+                    <div className={classes.actionCellIcons}>
```

Comment:
Why do you need a div?

Reply:
I wanted to add the class I previously added and it looks like with react `className` you can only add a single class nicely, with more you have to use interpolation or something like that?

Anyways, it's gone now, since I got rid of the need for extra styling